website/docs/components/app-header/partials/specifications/anatomy.md

```diff
@@ -0,0 +1,29 @@
+## Anatomy
+
+### Large
+
+![Anatomy of the large App Header](/assets/components/app-header/app-header-anatomy-large.png)
```

Comment:
[nit] "HomeLink" is missing a space between the words, like the other elements in the image. Should be "Home link"

Reply:
Got both of these updated!